local-test-operator.yml

```diff
@@ -17,26 +18,31 @@
     work_dir: "/tmp/operator-test"
     testing_bin_path: "{{ work_dir }}/bin"
     current_channel: '' # Added to avoid a potential bug with undefined variables
-    distro_type: "{{ DISTRO_TYPE | default('') }}"
-  
+    distro_type: "{{ 'upstream' if run_upstream else '' }}"
+
+  environment:
+    PATH: "{{ ansible_env.PATH }}:{{ ansible_env.HOME }}/.local/bin"
+    when: run_upstream|bool
+
+  pre_tasks:
+    - setup:
+      tags:
+        - always
+      when: run_upstream|bool
+
   tasks:
     - set_fact:
-        distro_type: "{{ DISTRO_TYPE | default('') }}"
```

Comment:
will be missing in roles?

Reply:
This is not needed, because `distro_type` is variable and not set as fact